src/Network/Network/Cortex/VpnConnection/NewAzVpnSiteLinkConnectionCommand.cs

```diff
@@ -0,0 +1,139 @@
+﻿namespace Microsoft.Azure.Commands.Network
+{
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.Network.Models;
+    using System;
+    using MNM = Microsoft.Azure.Management.Network.Models;
+    using System.Security;
+    using Microsoft.WindowsAzure.Commands.Common;
+
+    [Cmdlet(
+        VerbsCommon.New,
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "VpnSiteLinkConnection",
+        SupportsShouldProcess = false),
+        OutputType(typeof(PSVpnSiteLinkConnection))]
+    public class NewAzVpnSiteLinkConnectionCommand : NetworkBaseCmdlet
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "Name")]
```

Comment:
Please clarify, which resource's name it is

Reply:
done